### DISCUSSION TITLE
IndexedDB adapter.

### DISCUSSION THREAD

**Discussion by ericvida** [maxnowack/signaldb#379] [General]
SignalDB looks great. It's exactly I'm looking for for internal projects. Could you make an adapter for IndexedDB. I see mention of it as possible, but I'm not very confident using IndexedDB to make a good adapter, thus I'm looking into libraries like SignalDB. ^_^

**Comment by maxnowack** [comment]
An IndexedDB adapter is definitely possible and should be easy to develop. I'll look into it when I find the time. It would be good to better understand your use case for an IndexedDB adapter, and also why you prefer IndexedDB over LocalStorage when it comes to using it with SignalDB?

**Reply by harrisonlo** [reply]
@maxnowack IndexedDB allows storing a much larger amount of data https://softwareengineering.stackexchange.com/questions/219953/how-is-localstorage-different-from-indexeddb

**Reply by maxnowack** [reply]
There is now an adapter for IndexedDB available: https://signaldb.js.org/reference/indexeddb/
